protected/humhub/modules/admin/controllers/ApprovalController.php

```diff
@@ -121,4 +163,73 @@ public function actionDecline($id)
             'approveFormModel' => $model
         ]);
     }
+
+    /**
+     * @return string|\yii\console\Response|\yii\web\Response
+     * @throws HttpException
+     * @throws \Throwable
+     * @throws \yii\base\InvalidConfigException
+     */
+    public function actionBulkActions()
+    {
+        /** @var string $action */
+        $action = Yii::$app->request->post('action');
+        /** @var array $usersId */
+        $usersId = Yii::$app->request->post('ids');
+
+        if (!$action) {
+            $this->view->error(Yii::t('AdminModule.controllers_ApprovalController', 'Please select an action (approve or decline)'));
+            return $this->redirect(['index']);
+        }
+
+        if (!$usersId) {
+            $this->view->error(Yii::t('AdminModule.controllers_ApprovalController', 'Please select some users (tick the checkboxes)'));
```

Comment:
- Category: AdminModule.base
- Message: `No users were selected.`

Reply:
We should disable the buttons until users where selected.

Reply:
Done